src/HasDependencies.php

```diff
@@ -92,12 +92,14 @@ protected function applyRulesForChildFields($childField)
      */
     public function validateFields() {
         $availableFields = [];
-        foreach ($this->action()->fields() as $field) {
-            if ($field instanceof NovaDependencyContainer) {
-                $availableFields[] = $field;
-                $this->extractChildFields($field->meta['fields']);
-            } else {
-                $availableFields[] = $field;
+        if (!is_null($this->action()->fields())) {
```

Comment:
How would the action fields ever be null? That's not the method signature of the method that returns fields and is more likely a problem with not following that method signature over a bug. That being said, do you have an example of this being an issue?